packages/core/src/components/Tabs/Tabs.jsx

```diff
@@ -113,7 +152,9 @@ export class Tabs extends React.PureComponent {
           id={panelTabId(panel)}
           key={panel.key}
           onClick={this.handleTabClick}
+          onKeyDown={this.handleTabKeyPress}
           panelId={panel.props.id}
+          ref={(tab) => { this.tabs[panel.props.id] = tab; }}
```

Comment:
I believe you want the tab `id` here instead? That would be `panelTabId(panel)`. You could create a `const` for that so you only need to call the method once.